#    North(x)
#   /
#  /
# /______East(y)
# |
# |
# Donw (-z)
Header header
int32 ts
float32 x
float32 y
float32 z 
